game/scripts/npc/heroes/dark_willow.txt

```diff
@@ -9,7 +9,7 @@
     "Ability11"                                           "special_bonus_unique_dark_willow_1" // replaces special_bonus_unique_dark_willow_5
 
     //"Ability12"                                           "special_bonus_unique_dark_willow_4"
-    "Ability13"                                           "special_bonus_attack_damage_40" // replaces special_bonus_unique_dark_willow_7
+    "Ability13"                                           "special_bonus_attack_damage_30" // replaces special_bonus_unique_dark_willow_7
```

Comment:
worth considering replacing the bonus shadow realm duration talent entirely?

Reply:
I don't have ideas for new talents at the moment.